metadata_service/proxy/atlas_proxy.py

```diff
@@ -375,6 +377,23 @@ def get_table(self, *, table_uri: str) -> Table:
                     )
                 )
 
+            reports = []
+            for report in attrs.get("reports") or list():
+                LOGGER.info("Getting matadata for table Report GUID:{}".format(report.get("guid")))
+                report_entity = self._driver.entity_guid(report.get("guid"))
```

Comment:
you could get all the reports with one request instead of doing per-report-request

Reply:
You're right, I'll use entity_bulk